README.md

```diff
@@ -63,8 +68,10 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
-```
+
+Scope deals with where functions and variables can be accessed on the page.
+If variables are located inside of a scope they can only bee accessed there.
+If they are in the global scope, they can be accessed anywhere.
 
```

Comment:
:+1:  It helps to remember the 4 helpful rules of scope: 
1. Variables defined without the var keyword are always placed in the global scope.
2. Variables that are defined using the var keyword are created in the current local scope.
3. Every function and only functions create a new local scope.
4. The current scope includes all outer (enclosing) scopes.